GettingStartedwithAnsibleusingGNS3.md

```diff
@@ -0,0 +1,283 @@
+#	netmask 255.255.255.0
+#	gateway 192.168.0.1
+#	up echo nameserver 192.168.0.1 > /etc/resolv.conf
+
+# DHCP config for eth0
+auto eth0
+iface eth0 inet dhcp
+```
+4. Power-on the container and launch its console
+5. Once powered-on, the container should automatically get an IP address from the NAT cloud via DHCP with default subnet of 192.168.122.0/24
+>NOTE: This is the reason why we have assigned an IP address from 192.168.122.0/24 subnet to the EXOSVM.
+6. Ping the EXOSVM and make sure that it is reachable from the Automation container
+7. Run command ```ansible --version``` to check the currently installed version of Ansible
+
+## System Setup
+
+Ideally, there is no need to install ansible on Network Automation Docker Container as it already comes pre-installed with it. But for those who are using their own standalone Linux Machines can go though the below section to install the same.
+
+### Installing Ansible
+
+1. Run the below commands to install the required files/packages:
+
+```sh
+root@NetworkAutomation-1:~# apt-get update
+root@NetworkAutomation-1:~# apt-get install git
+root@NetworkAutomation-1:~# git clone https://github.com/ansible/ansible.git --recursive
+```
+2. Do a 'ls' to make sure that ansible folder is successfully cloned. Sample command and its output is below:
+
+```sh
+root@NetworkAutomation-1:~# ls
+ansible
+```
+
+3. One can periodically pull the latest developments from Ansible's Github Repository by running the below commands:
+
+```sh
+root@NetworkAutomation-1:~# cd ansible
+root@NetworkAutomation-1:~#git pull
+```
+
+### Setting up the Virtual Environment
+
+The main purpose of Python virtual environments is to create an isolated environment for Python projects. This means that each project can have its own dependencies, regardless of what dependencies every other project has. It is not really necessary to create a virtual environment but it is recommended to do so. Visit this [page] for details.
+
+1. Run below commands to setup the virtual environment:
+
+```sh
+root@NetworkAutomation-1:~# apt-get install virtualenv
+root@NetworkAutomation-1:~# cd ansible
+root@NetworkAutomation-1:~/ansible# virtualenv venv
+```
+2. Do a 'ls' while being under 'ansible' folder and make sure that a 'venv' folder is successfully created. Sample command and its output is below:
+
+```sh
+root@NetworkAutomation-1:~/ansible# ls
+CODING_GUIDELINES.md  bin                       hacking           shippable.yml
+COPYING               changelogs                lib               test
+MANIFEST.in           contrib                   licenses          tox.ini
+MODULE_GUIDELINES.md  docs                      packaging         venv
+Makefile              docsite_requirements.txt  requirements.txt
+README.rst            examples                  setup.py
+```
+
+3. Activate the 'venv' virtual environment
+
+```sh
+root@NetworkAutomation-1:~#source ./ansible/venv/bin/activate
+root@NetworkAutomation-1:~#source ./ansible/hacking/env-setup
```

Comment:
Should be space between PS1 prompt (`~#`) and the commands above.

Reply:
done